Makefile

```diff
@@ -62,6 +64,15 @@ clean:
 	$(CARGO) clean
 	make -C seabios clean
 
+.PHONY: dev-init
+dev-init: install-git-hooks
+
+.PHONY: install-git-hooks
+install-git-hooks: $(git_hooks)
+
+$(git_hooks): $(git_hooks_src)
+	cp $< $@
```

Comment:
Would a link be appropriate here? So you wouldn't need to run `dev-init` again if we were to update the script?

Reply:
fair point. done. 

I suppose we could also make the dev-init target a dependency for something else to ensure it remains up-to-date. Maybe test? Probably not worth doing for now though.